app/views/export/_invitation.html.erb

```diff
@@ -0,0 +1,50 @@
+<tr>
+<td>
+  <%= invitation.invitee.try(:full_name) %> <%= invitation.email %>
+</td>
+<td>
+  <%if invitation.decision.major_version%>
+    v<%= invitation.decision.major_version %>.<%= invitation.decision.minor_version %>
+    <%= invitation.decision.verdict.try(:humanize) %>
+  <% end %>
+</td>
+<td>
+  <%if !invitation.pending? %>
+    <%if invitation.invited? %>
+      Invited <%= invitation.invited_at%>
+    <% else %>
+      <%if invitation.accepted? %>
+        <%if ReviewerReport.for_invitation(invitation) %>
+          <%if ReviewerReport.for_invitation(invitation).status == 'completed' %>
+            Completed <%= ReviewerReport.for_invitation(invitation).datetime %>
+          <% else %>
+            <%if ReviewerReport.for_invitation(invitation).due_datetime.due_at %>
+              Review due <%= ReviewerReport.for_invitation(invitation).due_datetime.due_at %>
+            <% else %>
+              Review pending
+            <% end %>
+            <%if invitation.actor && !invitation.is_accepted_by_invitee %>
+              Accepted by <%= invitation.actor.full_name %>
+            <% end %>
+          <% end %>
+        <% else %>
+          Accepted <%= invitation.accepted_at%>
+        <% end %>
+      <% else %>
+        <%if invitation.declined? %>
+          Declined <%= invitation.declined_at%>
+        <% else %>
+          <%if invitation.rescinded? %>
+            Rescinded <%= invitation.rescinded_at%><% end %>
+        <% end %>
+      <% end %>
+    <% end %>
+  <% end %>
+</td>
+<td>
+  <%if invitation.declined? %>
+    <p><%= invitation.decline_reason%></p>
+  <% end %>
+  <%= invitation.body %>
+</td>
+</tr> 
```

Comment:
Would we want to add things like `decline_reason`, `invitee_role` and `information`? https://github.com/Aperta-project/Aperta/blob/develop/db/schema.rb#L478-L502

Reply:
I want to mirror the logic in this template, nothing more: https://github.com/Aperta-project/Aperta/blob/develop/client/app/pods/components/invitation-detail-row/template.hbs

I considered it and did at first include `decline_reason`. It only has meaning when the invitation is declined.  The template logic presents `decline_reason` when it's appropriate.